src/components/formatters/include/formatters/CFormatterJsonSDLRPCv2.h

```diff
@@ -151,11 +151,14 @@ inline bool CFormatterJsonSDLRPCv2::fromString(
   bool result = true;
 
   try {
+    Json::CharReaderBuilder builder;
+    const std::unique_ptr<Json::CharReader> reader(builder.newCharReader());
     Json::Value root;
-    Json::Reader reader;
+    const int strLength = static_cast<int>(str.length());
 
     namespace strings = ns_smart_device_link::ns_json_handler::strings;
-    bool result = reader.parse(str, root);
+    bool result =
+        reader->parse(str.c_str(), str.c_str() + strLength, &root, nullptr);
 
     if (true == result) {
```

Comment:
@ydementieiev if(result) ?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)